### ISSUE TITLE
State is not announced  whether the item is collapsed or elapsed.

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#760]
**Describe the bug**
Users who rely on screen readers will affect to navigate if state is not announced by the Narrator for combo box. The state is not announced whether the item is collapsed or elapsed.  

**To Reproduce**
Steps to reproduce the behaviour:
Step 1: Open Graph Explorer URL in Anaheim dev  Edge browser.
Step 2: Navigate to the left pane.
Step3: Navigate to the Controls present under "Search sample queries".
Step 4: Observe the issue.

**Expected behaviour**
The state should be announced whether the item is collapsed or elapsed.

**Desktop (please complete the following information):**
 - OS: OS build Version 2004 (OS Build 20170.1000)
 - Browser: Anaheim Edge Dev Version: 87.0.634.0 (Official build) dev (64-bit)
 - Screen Reader: Narrator 
[AB#7472](https://microsoftgraph.visualstudio.com/0985d294-5762-4bc2-a565-161ef349ca3e/_workitems/edit/7472)

**Comment by ElinorW** [comment]
At the moment, state is  announced when one navigates to the caret icon displayed 
![image](https://user-images.githubusercontent.com/18407044/107003963-a1ae9900-679e-11eb-8103-c6a9192bdc52.png)

**Comment by ddyett** [comment]
@ElinorW what's the next step to this.  Are we waiting for someone to address something?

**Comment by ElinorW** [comment]
@ddyett at that time, there was no implementation on Fluent UI yet... but I am revisiting the issue and looking into other options.

**Comment by adhiambovivian** [comment]
@ElinorW  let's revisit this one during our next milestone.

**Comment by ElinorW** [comment]
@adhiambovivian This was fixed with the upgrade to Fluent UI. I forgot to close this.
